consoleme/celery_tasks/celery_tasks.py

```diff
@@ -573,8 +573,26 @@ def cache_policies_table_details() -> bool:
     if all_s3_errors:
         s3_errors = json.loads(all_s3_errors)
 
+    allowed_tags = config.get("roles.allowed_tags", {})
+    allowed_arns = config.get("roles.allowed_arns", [])
     for arn, role_details_j in all_iam_roles.items():
         role_details = ujson.loads(role_details_j)
+        role_details_policy = ujson.loads(role_details.get("policy"), {})
+
+        allowed = True
+
+        actual_tags = {
+            k: v for d in role_details_policy.get("Tags") for k, v in d.items()
```

Comment:
I think this line will throw an exception if `role_details_policy` doesn't contain `Tags`. So best to add a conditional around this.

Reply:
thx, adding `if "Tags" in role_details_policy:`